### PR TITLE
[5.1] Update bump script also to update package-lock.json

### PR DESCRIPTION
**Pull request by HLeithner**
Pull Request for Issue #43743 .

### Summary of Changes
Also update the package-lock.json on version bump

### Testing Instructions
php build/bump.php -v 5.2.3

### Actual result BEFORE applying this Pull Request
package.json is updated
package-lock.json is not updated

### Expected result AFTER applying this Pull Request
package.json is updated
package-lock.json is updated

### CODE REVIEW

**Comment by LadySolveig** [comment]
Thanks @HLeithner

File: build/bump.php [hunk 1]
```diff
@@ -248,12 +251,14 @@ function usage($command)
 }
 
 // Updates the version in the package.json file.
-if (file_exists($rootPath . $packageJsonFile)) {
-    $package          = json_decode(file_get_contents($rootPath . $packageJsonFile));
-    $package->version = $version['release'];
+foreach($packageJsonFiles as $packageJsonFile) {
```
Comment (Quy) [inline build/bump.php]:
```suggestion
foreach ($packageJsonFiles as $packageJsonFile) {
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on eba833dc71b78d0809b571ed96341074465e970a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43801">issues.joomla.org/tracker/joomla-cms/43801</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on eba833dc71b78d0809b571ed96341074465e970a<br /><br />by code review



Thanks!!!!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43801">issues.joomla.org/tracker/joomla-cms/43801</a>.</sub>

**Comment by brianteeman** [comment]
(I got a php fatal error on line 313 BUT i ran it on windows and I seem to recall the script will not run on windows and that part of the file is not part of this PR and the parts in this PR all completed correctly)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43801">issues.joomla.org/tracker/joomla-cms/43801</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43801">issues.joomla.org/tracker/joomla-cms/43801</a>.</sub>

**Comment by HLeithner** [comment]
I tested it and it worked on linux, also line 313 is only a preg_replace so shouldn't fail

**Comment by LadySolveig** [comment]
Thank you @brianteeman and @richard67 for testing!

File: build/bump.php [hunk 2]
```diff
@@ -248,12 +251,14 @@ function usage($command)
...
-    $package->version = $version['release'];
+foreach ($packageJsonFiles as $packageJsonFile) {
+    if (file_exists($rootPath . $packageJsonFile)) {
+        $package          = json_decode(file_get_contents($rootPath . $packageJsonFile));
+        $package->version = $version['release'];
 
-    // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
-    file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)));
+        // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
+        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)));
```
Comment (dgrammatiko) [inline build/bump.php]:
```suggestion
        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)) . '/n');
```

The Code Style for JS/JSON requires an empty line at the end of the file

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

File: build/bump.php (hunk 2 above)
Comment (richard67) [reply build/bump.php]: That’s right.

**Comment by LadySolveig** [comment]
Thanks @dgrammatiko

File: build/bump.php [hunk 3]
```diff
@@ -248,12 +251,14 @@ function usage($command)
...
-    $package->version = $version['release'];
+foreach ($packageJsonFiles as $packageJsonFile) {
+    if (file_exists($rootPath . $packageJsonFile)) {
+        $package          = json_decode(file_get_contents($rootPath . $packageJsonFile));
+        $package->version = $version['release'];
 
-    // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
-    file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)));
+        // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
+        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)) . '/n');
```
Comment (dgrammatiko) [inline build/bump.php]:
```suggestion
        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)) . "\n");
```

Comment (dgrammatiko) [reply build/bump.php]: Oops I've messed it up, now should be fine

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

File: build/bump.php (hunk 3 above)
Comment (LadySolveig) [reply build/bump.php]: Thanks

**Pull request merged**